packages/release/opt-cni-bin.mount

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CNI Plugin Directory (/opt/cni/bin)
+Conflicts=umount.target
+RequiresMountsFor=/opt
+Before=local-fs.target umount.target
+
+[Mount]
+What=overlay
+Where=/opt/cni/bin
+Type=overlay
+Options=noatime,nosuid,nodev,lowerdir=/usr/libexec/cni/bin,upperdir=/opt/cni/upper,workdir=/opt/cni/work,context=system_u:object_r:local_t:s0
```

Comment:
Would it be worth adding another directory here to contain the upperdir and workdir to hide them/make it more obvious that they're an implementation detail of the overlay mount?  Something like `/opt/cni/.overlay/upper` and `/opt/cni/.overlay/work`?

Reply:
I opted to use `/var/lib/cni-plugins` for the overlay directories, partly to match the treatment of `/var/lib/kernel-devel` (which I also adjusted along these lines), and partly to guard against cases where pods might be mounting in `/opt/cni` and get confused by the new directories.